api/swagger.yaml

```diff
@@ -167,7 +212,7 @@ paths:
                       properties:
                         exposedPort:
                           description: The port inside the container.
-                          type: uint
+                          type: number
```

Comment:
Is this an intended change here?

Reply:
Intended yes, but not as correct as it should be

Reply:
Ah, I see. Thanks for pointing that out!